features/payment/managing_payment_methods/browsing_payment_methods.feature

```diff
@@ -0,0 +1,16 @@
+@managing_payment_methods
+Feature: Browsing payment methods
+    In order to see all payment methods in the store
+    As an Administrator
+    I want to be able to browse payment methods
+
+    Background:
+        Given the store has a payment method "Offline" with a code "OFF"
```

Comment:
If there's only one scenario, can we move the first two steps of the background into the scenario?